backend/src/main/java/de/neuefische/backend/controller/todoController.java

```diff
@@ -29,4 +29,9 @@ public TodoElement getTodoById(@PathVariable String id){
         return service.getTodoById(id);
     }
 
+    @PutMapping("/{id}")
+    public TodoElement updateTodo(@RequestBody TodoElement todoElement){
```

Comment:
no need for pathvariable?
What happens when u use postman which transmitted an object with an unknown Id? i guess then its just getting created...and this is fine?